### PR TITLE
merging changes into kamalio_pcscf.cfg

### PR DESCRIPTION
**Pull request by NUCLEAR-WAR**
I added your changes to this file and test it with my setup.
THE NF Register works perfect.

The NGHTTP2 module needs key and Cert file in pem format to load, I generated a self signed ones, currently replying 204 only on the subscription URI that ends with /terminate.

### CODE REVIEW

File: pcscf/kamailio_pcscf.cfg [hunk 1]
```diff
@@ -475,7 +484,177 @@ modparam("statistics", "variable", "register_failed")
...
+       nghttp2_reply_header("accept", "application/json");
+       nghttp2_reply_header("accept", "application/problem+json");
+       nghttp2_reply_header("Content-Type", "application/json");
+       nghttp2_reply_header("Server", "PCSCF");       
+       nghttp2_reply("404", "Not Found");
+   }
+}
+
+
+##################################################################
```
Comment (herlesupreeth) [inline pcscf/kamailio_pcscf.cfg:542]: please remove this part. I will cherry-pick my commits on top

File: pcscf/kamailio_pcscf.cfg [hunk 2]
```diff
@@ -475,7 +484,177 @@ modparam("statistics", "variable", "register_failed")
...
+modparam("http_client", "httpredirect", 1)
+modparam("http_client", "useragent", "PCSCF")
+modparam("http_client", "maxdatasize", 64535)
+modparam("http_client", "connection_timeout", 2)
+modparam("http_client", "keep_connections", 1)
+modparam("http_client", "response_headers", 1)
+modparam("http_client", "query_result", 1)
+modparam("http_client", "httpcon", "pcfapi=>http://172.22.0.27:7777")
+
+modparam("nghttp2", "listen_addr", "172.22.0.21")
```
Comment (herlesupreeth) [inline pcscf/kamailio_pcscf.cfg:507]: i will replace these part later when I cherry-pick my commits on top of your changes

File: pcscf/kamailio_pcscf.cfg [hunk 3]
```diff
@@ -475,7 +484,177 @@ modparam("statistics", "variable", "register_failed")
...
+       nghttp2_reply_header("accept", "application/problem+json");
+       nghttp2_reply_header("Content-Type", "application/json");
+       nghttp2_reply_header("Server", "PCSCF");       
+       nghttp2_reply("404", "Not Found");
+   }
+}
+
+
+##################################################################
+## 5G N5 Registring PCSCF to SCP   Code added by Supreeth Herle ##
```
Comment (herlesupreeth) [inline pcscf/kamailio_pcscf.cfg:543]: honestly, not a big fan of having my name in the code. So remove it.

**Review by herlesupreeth — CHANGES_REQUESTED** [review]
There is something strange about this PR. Its missing my changes so I would suggest rebasing the changes in exp_5g_ims_pyhss branch and applying your changes on top

File: pcscf/kamailio_pcscf.cfg (hunk 2 above)
Comment (NUCLEAR-WAR) [reply pcscf/kamailio_pcscf.cfg:507]: that's would be nice

**Comment by NUCLEAR-WAR** [comment]
can you check it now ?

**Comment by herlesupreeth** [comment]
Actually we are adding twice (loadmodule "nghttp2" etc) but its okay. Once merged I will fix it. Thanks again.

**Pull request merged**